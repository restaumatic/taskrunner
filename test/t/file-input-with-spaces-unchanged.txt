# no toplevel

echo lol > 'some input.txt'
git init -q
git add 'some input.txt'

go() {
  taskrunner -n mytask bash -e -c '
    snapshot "some input.txt"
    echo "Expensive computation"
  '
}

go
go
