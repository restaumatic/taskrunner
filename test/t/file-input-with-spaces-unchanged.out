-- output:
[mytask] stdout | Expensive computation
